src/wp-includes/html-api/class-wp-html-processor-state.php

```diff
@@ -356,17 +356,6 @@ class WP_HTML_Processor_State {
 	 */
 	public $insertion_mode = self::INSERTION_MODE_INITIAL;
 
-	/**
-	 * Context node initializing fragment parser, if created as a fragment parser.
-	 *
-	 * @since 6.4.0
-	 *
-	 * @see https://html.spec.whatwg.org/#concept-frag-parse-context
-	 *
-	 * @var [string, array]|null
-	 */
-	public $context_node = null;
```

Comment:
I have an impression that this property should remain forever, but it should be marked as `@deprecated` for compatibility reasons. `WP_HTML_Processor_State` isn't annotated as allowing dynamically created properties, like in other places, example:

https://github.com/WordPress/wordpress-develop/blob/c344148315693d6592283b67e232e8609169c2f9/src/wp-includes/class-wp-block.php#L15-L16

However, it's my suspicion that needs to be validated.

Reply:
Thank you, I was curious about this.

I'll restore the property.